### ISSUE TITLE
Input Addon doesn't have a focus ring [accessibility]

### ISSUE THREAD

**Issue by FatahChan** [ant-design/ant-design#55237]
### Reproduction link

[![Edit on StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/edit/psfvqgcy?file=src%2Fdemo.tsx)

### Steps to reproduce

click on the second input, then press tab, focus ring disappear, press enter the select options opens up

### What is expected?

focus ring should be present

### What is actually happening?

focus ring should be disappear

| Environment | Info |
| --- | --- |
| antd | 5.27.4 |
| React | 19.2 |
| System | MacOS |
| Browser | Chrome |

**Comment by FatahChan** [comment]
@desu antd is suppressing it

**Comment by MadCcc** [comment]
Try Space.Compact?
Input addon is not recomended in 5.x

**Comment by EmilyyyLiu** [comment]
请分配给我处理吧

**Comment by EmilyyyLiu** [comment]
请分配给我处理
